R/format_loanbook_st.R

```diff
@@ -82,7 +82,7 @@ format_loanbook_st <- function(data,
       scenario = .data$metric,
       scenario_geography = .data$region,
       ald_sector = .data$sector,
-      company_name = .data$name_ald
+      company_name = .data$name_abcd
```

Comment:
what is the consequence on backward compatibility of this change?

Reply:
none, if we say that 2021Q4 data is a prerequisite for loan books. Which we do implicitly by the selection of files that are required for the loan book preparation. Otherwise users will have to rename said columns manually in the input data